grml-debootstrap

```diff
@@ -1306,6 +1311,15 @@ mkfs() {
   fi
 
   if [ -n "$MKFS" ] ; then
+
+    if [ -n "$ARM_EFI_TARGET" ] ; then
+      einfo "Running mkfs.vfat $MKFS_OPTS on $ARM_EFI_TARGET"
+      mkfs.vfat -n "EFI" $MKFS_OPTS "$ARM_EFI_TARGET"
```

Comment:
FYI, the inclusion of `$MKFS_OPTS` was causing an error for me after merging this branch into the latest master:

```
* Running mkfs.vfat  -O ^metadata_csum_seed on /dev/mapper/loop15p1
mkfs.fat 4.1 (2017-01-24)
mkfs.vfat: invalid option -- 'O'
```

Removing `$MKFS_OPTS` from this line got me past it. At a glance, all the $MKFS_OPTS seem to only apply to `mkfs.ext4`, so it might not be needed here at all.

Reply:
You're right, have pushed a commit which fixes this. Thanks!